todo

```diff
@@ -210,4 +228,11 @@ if __name__ == '__main__':
 
 	# Supress display after adding/deleting/clearing
 	if args.suppress is None or args.suppress is False:
-		printTodo(todo, ids, args.verbose, args.tag_main)
\ No newline at end of file
+		if args.command == 'lolcat':
+			if args.spread == 0.0:
+				args.spread = 3.0
+			args.mode = args.mode or detect_mode()
+			args.seed = random.randint(0, 256) if args.seed == 0 else args.seed
+			printTodo(todo, ids, args.verbose, args.tag_main, args, enable_lolcat=True)
+		else:
+			printTodo(todo, ids, args.verbose, args.tag_main, args)
```

Comment:
1. Would be cleaner to just call `printTodo(todo, ids, args, enable_lolcat=(args.command == 'lolcat'))` here. You should check and initialise `Lolcat` settings (`spread`, `mode`, and `seed`) inside the `Lolcat` class. `detect_mode` can be made a method of `Lolcat` as it won't be used anywhere else.

2. Check for `<=0` instead of `==0` when setting `spread` and `seed`.

3. Passing `args.verbose` and `args.tag_main` are redundant as you're passing `args` as a whole. Refactor the `printTodo` method to just take `args`.

Reply:
Refactored the code accordingly